apps/src/code-studio/pd/workshop_dashboard/components/date_picker.jsx

```diff
@@ -95,6 +95,7 @@ export default class DatePicker extends React.Component {
   static propTypes = {
     date: PropTypes.object,
     onChange: PropTypes.func.isRequired,
+    onBlur: ReactDatePicker.propTypes.onBlur,
```

Comment:
huh, this is a cool pattern. Should we update the other props that are passed directly through to ReactDatePicker to mirror this?

Reply:
:+1: I'd be in favor of that!  Or even something more ridiculous, calling out that we're just passing through loads of these:
```jsx
static propTypes = {
  date: PropTypes.object,
  onChange: PropTypes.func.isRequired,
  ..._.pick(ReactDatePicker.propTypes, [
    'onBlur',
    'minDate',
    'maxDate',
    'selectsStart',
    // ...
  ]),
};
```